### PR TITLE
Added instructions for using CyVerse VICE apps

### PR DESCRIPTION
**Pull request by MagicMilly**

### CODE REVIEW

**Review by dlebauer — CHANGES_REQUESTED** [review]
* The README should start with additional information about the purpose of this repository, and then have a section on "cleaning data using vice" or such
* it might help to add screenshots?
* I think @tyson-swetnam can cache the image so it doesn't take as long to load

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
 # terraref-datasets
-Repository for code and small datasets derived from the TERRA REF program
+
+For **GenoPhenoEnvo** collaborators: if you already have a [CyVerse](https://cyverse.org/) account, please follow these instructions to access the **reproducible data cleaning apps** for MAC Sorghum Seasons 4 and 6 and Kansas Sorghum Experiments. These apps and associated input data have not yet been made public. If you do not have app access, please make sure that you have a CyVerse account and send me your username via Slack or [e-mail](mailto:ejcain@arizona.edu).
+
```
Comment (dlebauer) [inline README.md:4]:
```suggestion

## Genomics data

https://datacommons.cyverse.org/browse/iplant/home/shared/terraref/genomics/derived_data/bap/resequencing/danforth_center/version1/hapmap

```

**Comment by dlebauer** [comment]
Maybe some of this info should go in the documentation repository (perhaps as a separate md) or at least link to this readme from the documentation readme https://github.com/genophenoenvo/documentation/blob/master/README.md

**Comment by MagicMilly** [comment]
Closing this PR because we've moved away from using the CyVerse app, so it is no longer updated, and even with instructions was difficult for collaborators to use.

**Pull request closed without merging**